task-definition.migration.json

```diff
@@ -83,6 +83,10 @@
                 {
                     "name": "SITE_URL",
                     "value": "https://app.posthog.com"
+                },
+                {
+                    "name": "PARALLEL_COHORTS",
+                    "value": "0"
```

Comment:
I'd set this down to 1 so that we can process them serially.
https://github.com/PostHog/posthog/blob/master/posthog/tasks/calculate_cohort.py#L23